content/apps/how-to-test-my-app.md

```diff
@@ -0,0 +1,64 @@
+# How to test my App?
+
+::: warning
+**For now, this feature is ONLY AVAILABLE on partner sandboxes Akeneo provides**
+:::
+
+This page explains how to enable the `developer mode` on your partner sandbox and how to create a test App to check that everything is working well. 
+
+## Step 1 - Enable the developer mode and the test App creation
+
+To enable the developer mode and the `Create a test App` button, you need to:
+1. Go to `Settings`, then `Roles`
+2. Choose the role you use for your user
+3. In the `Permissions` tab, scroll down and search for the `Developer mode` submenu
+4. Select `Manage test apps`
+5. Don't forget to save your modifications
+
+![Role permissions - developer mode](../img/apps/role-developer-mode.png)
+
+## Step 2 - Create a test App 
+
+![Create a test app button](../img/apps/create-a-test-app-button.png)
+
+To create a test App: 
+1. Go to `Connect`, then `Marketplace`
+2. On the top right corner, click on `Create a test App` 
+3. Fill in all the required information
+4. Then click on `Create`
+5. Copy/paste credentials in your app configuration file
+6. And click on `Done`
+7. Your test App appears on the Marketplace page
+
+![Test app creation](../img/apps/test-app-creation.png)
+
+
+## Step 3 - Connect your test App 
+
+![Test app on Marketplace](../img/apps/marketplace-with-test-app.png)
+
+Connect a test App is like connecting a published App. 
+
+1. Click on `Connect`
+2. Your App opens in a new tab of your browser
+3. Launch the connection process from your App
+4. Follow all the activation process steps, then `Confirm`
+5. Your test App is now connected with Akeneo PIM! 🔗
+
+::: info
+To know more about the step-by-step activation process, please read our article: [How to connect an App?](https://help.akeneo.com/pim/serenity/articles/how-to-connect-my-pim-with-apps.html#how-to-connect-an-app)
+:::
+
+
+## Step 4 - Access the connected App settings on Akeneo PIM
+
+Now, your App is connected you can enjoy all the available App features from the Akeneo PIM UI and test that your App is working well. 
+
+To access the settings of your connected App on the Akeneo PIM side, please go to `Apps`, then click on `Manage App`. 
+You can also open your App from Akeneo PIM UI, to do so, click on `Open app`. 
+
+![Connected test app on Apps](../img/apps/connected-test-app.png)
+
+::: info
+To know more about connected app management, please read our article: [Manage your Apps](https://help.akeneo.com/pim/serenity/articles/manage-your-apps.html)
+:::
```

Comment:
I think we should remove this part, IMHO, it's not really interesting, in the test app context.
It feel like "noise".
In my opinion, your document would be great without it.